node/node.go

```diff
@@ -1554,7 +1554,7 @@ func (app *App) grpcService(svc grpcserver.Service, lg log.Log) (grpcserver.Serv
 		app.grpcServices[svc] = service
 		return service, nil
 	case v2alpha1.Activation:
-		service := v2alpha1.NewActivationService(app.apiDB)
+		service := v2alpha1.NewActivationService(app.apiDB, app.cachedDB, types.ATXID(app.Config.Genesis.GoldenATX()))
```

Comment:
I thought `app.cachedDB` should not be used in the API any more? The method `MaxHeightAtx` is just `atxs.GetIDWithMaxHeight(db, types.EmptyNodeID, atxs.FilterAll)` and `GetAtx` is also just `atxs.Get(db, id)`.